### DISCUSSION TITLE
Hewbrew: Location of shortcut indicator (&)

### DISCUSSION THREAD

**Discussion by buhtz** [WeblateOrg/weblate#10940] [General]
I know nothing about Hebrew. So If one of you know Hebrew native speakers it would be great if you could point them here to that question.

![grafik](https://github.com/WeblateOrg/weblate/assets/11861496/9fe8178a-045a-42ac-9f06-ce9edc0cf4e1)

The first geen arrow points to the letter `&` in the translated string on Weblate. The `&` is on the right "end" of that string. Because Hewbrew is a right-to-left language (and Weblate seems to understand it) I would say the `&` indicates that the Hebrew letter `א` (second from right) is the shortcut letter that can be used with ALT key in the GUI.
Am I right so far?

But looking into the raw po-file I am confused. There the `&` is located on the other site of translated string.

But looking into the GUI the translation and shortcut letter position seems OK.
![grafik](https://github.com/WeblateOrg/weblate/assets/11861496/1c87409b-8ab2-4f1c-a519-691d603ce04a)

So why is it "wrong" in the raw po-file?

**Comment by nijel** [comment]
In both Weblate and the application, the text is marked as RTL so is whole string is rendered RTL including the first character which is `&` - so it is on the right. In console, it is inside LTR text, so the rendering differs - it renders everything up to '&' as LTR and after it switches to RTL because of RTL letters.

To put it in other words - `&` is physically the first letter, if your current rendering is RTL it is rendered on the right side, if LTR on the left side of the string.
